rtdata/languages/default

```diff
@@ -2997,6 +2998,7 @@ TP_LOCALLAB_AVOID;Avoid color shift
 TP_LOCALLAB_AVOIDCOLORSHIFT_TOOLTIP;Fit colors into gamut of the working color space and apply Munsell correction (Uniform Perceptual Lab). Default: Munsell only.\n\nMunsell only: Fixes Lab mode hue drifts due to non-linearity when chromaticity is changed (Uniform Perceptual Lab).\nLab: Applies a gamut control in relative colorimetric. Munsell is then applied.\nXYZ Absolute: Applies gamut control in absolute colorimetric. Munsell is then applied.\nXYZ Relative: Applies gamut control in relative colorimetric. Munsell is then applied. The result is not the same as Lab.
 TP_LOCALLAB_AVOIDMUN;Munsell correction only
 TP_LOCALLAB_AVOIDMUN_TOOLTIP;Munsell correction always disabled when Jz or CAM16 is used.
+TP_LOCALLAB_AVOIDNEG;Avoid negatives values
```

Comment:
```suggestion
TP_LOCALLAB_AVOIDNEG;Avoid negative values
```

There could be confusion about what this does. Some people can read this and think it will prevent the spot from creating negative values. Maybe something like "Pre-filter negative values" would work?

Reply:
You don't think there will be confusion about what it does?

Reply:
After re-reading, I think you are true. I change for
"Pre-filter zero and negative values"